src/style.css

```
@@ -0,0 +1,137 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
```

Скоріш за все, скажуть, що Roboto треба взяти в кавички ("")
```suggestion
  font-family: Roboto, sans-serif;
```
```suggestion
  font-family: "Roboto", sans-serif;
```